### ISSUE TITLE
Suggestion: delay autosearch for items with poe.ninja cached price

### ISSUE THREAD

**Issue by dusty071** [Kyusung4698/PoE-Overlay#558]
I was just sorting my stash and was looking for items worth selling so i used Alt+D:
- 15 times for essences
- 10 times for scarabs
- 5 times for currencies
- 20 times for uniques
- 20 times for div cards
- 10 times for maps
- ...... you get it

All the while only looking at the poe.ninja price at the top and trying to spot something worth selling. And i noticed PoeOverlay automatically starts searching for live item listings even though i didnt need or want it to. Seeing how GGG has problems with their trade API because of tools pinging their servers too frequently i thought it might be sensible for your PoeOverlay tool to not immediately do a live autosearch for items that already have a poe.ninja price, but would instead wait for the user to trigger it in a similar way that we do when changing item mods. That could be the default setting with the option to set it back to the functionality that it has right now.

Just a thought.

**Comment by TrueEdge** [comment]
Yes please. It happens so often to me. I press ctrl+D on an item and then select the mods i want and every time i click on a mod it instantly starts to search again, then i click on the next mod and next search triggers. And so i get fast blocked from pathofexile.com trade for like 5 mins and can't live search. Would be nice to add a option to disable the autosearch or a second command for a search which needs a confirmation to actually start the search for the mods.
